### PR TITLE
send a disable message to disable puppet

### PR DESCRIPTION
**Pull request by nleib**
added an ability to send a "disabled_message" when using puppet.disable

### CODE REVIEW

File: salt/modules/puppet.py [hunk 1]
```diff
@@ -222,7 +222,7 @@ def enable():
     return False
 
 
-def disable():
+def disable(message=None):
     '''
     .. versionadded:: 2014.7.0
```
Comment (jfindlay) [inline salt/modules/puppet.py]: Will you add here
```
message
    disable message to send to puppet
    .. versionadded:: 2015.5.2
```

**Comment by jfindlay** [comment]
@nleib, will you also fix the [lint error](https://jenkins.saltstack.com/job/salt-pr-lint-n/5757/violations/file/salt/modules/puppet.py/)?  Thanks.

**Comment by nleib** [comment]
@jfindlay yes I will work on it again, thanks

**Comment by nleib** [comment]
Hey @jfindlay im not sure why the checks are failing. not sure it has to do anything with my commit.
can you please assist?

**Comment by jfindlay** [comment]
@nleib, that is an unrelated test failure.

**Pull request merged**